ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -756,18 +758,20 @@ private final class OracleQueries(tablePrefix: String) extends Queries(tablePref
   protected[this] override def initDatabaseDdls =
     super.initDatabaseDdls ++ Seq(stakeholdersView, stakeholdersIndex)
 
-  protected[http] override def version()(implicit log: LogHandler): ConnectionIO[Option[Int]] = {
+  protected[http] override def version()(implicit log: LogHandler): ConnectionIO[Option[Int]] =
```

Comment:
Presumably this was the other bug. What did the code actually do? Return `None` in all cases making `createSchema=false` always fail since the schema doesn’t exist? How do any of our tests work in that case?